front/app/containers/Admin/projects/all/Lists/ModeratorProjectList.tsx

```diff
@@ -61,13 +42,33 @@ const ModeratorProjectList = memo(() => {
               return (
                 !isProjectFoldersEnabled ||
                 (!adminPublication.relationships.parent.data && (
-                  <Row
-                    key={index}
-                    id={adminPublication.id}
-                    isLastItem={index === rootLevelAdminPublications.length - 1}
-                  >
-                    {adminPublicationRow(adminPublication)}
-                  </Row>
+                  <>
+                    {adminPublication.publicationType === 'project' && (
+                      <Row
+                        key={index}
+                        id={adminPublication.id}
+                        isLastItem={
+                          index === rootLevelAdminPublications.length - 1
```

Comment:
Minor and probably just an opinion. We could move this into a const above and name it `isLastItem` making this piece easy to read and reusing the last item logic